app/graphql/resolvers/concerns/collection.rb

```diff
@@ -20,8 +20,8 @@ module Collection
         end
       end
 
-      def resolve(**kwargs)
-        filters = filter_list(**kwargs)
+      def resolve(**)
```

Comment:
What a lovely shortcut.

Reply:
You don't like it? :trollface:

Reply:
Quite the opposite. I've meant what I've said. :slightly_smiling_face:

Reply:
Wait for the other goodies we will have with 3.3 😉